decidim-conferences/db/migrate/20221116084952_add_weight_to_conferences.rb

```diff
@@ -0,0 +1,5 @@
+class AddWeightToConferences < ActiveRecord::Migration[6.1]
+  def change
+    add_column :decidim_conferences, :weight, :integer, null: false, default: 0
```

Comment:
Please note that you also need to add it for consultations if you want to have a consistency across all the participatory spaces.

Reply:
I see that's already on #10062, so I'm fine with that approach as you need it on your PR so the build passes. 

I'm not so sure how to handle the duplicated conferences weight migration, as we both require it to pass our tests, but we need to be careful to not merge both of these at the same time. So, whatever PR is migrated first, the other will need to remove this migration and merge with develop. cc @decidim/maintainers

Reply:
@andreslucena I see 3 different solutions: 
- Merge my branch into yours ( so we have the same migration name)
- I could drop my PR, and apply the patch on your end. 
- One of us could set the branch in question as draft until the other is merged.
- We patch the migration files to use the rails 6.1 syntax: 
```
add_column :decidim_conferences, :weight, :integer, null: false, default: 0, if_not_exists: true
```

Reply:
I think those should be handled separately if we want to add the weight to the other components.